codalab/lib/worksheet_util.py

```diff
@@ -1019,6 +1020,8 @@ def flush_worksheets():
                     current_schema_ids.append(item_id)
                     current_schema_name = name
                     schemas[name] = current_schema = []
+                    raw_to_block.append((len(blocks) - 1 + len(current_schema_ids), 0))
```

Comment:
Need a comment on what this calculation means, and maybe we can name the raw_to_block_has_append to appended_<what the calculation represents>

Reply:
Changed the variable name and added comments.

Reply:
I think appended_schema_blocks_index would be better cc @epicfaace

Reply:
I changed the name to `appended_schema_blocks_index`